src/SFA.DAS.Reservations.Web/Controllers/ProviderReservationsController.cs

```diff
@@ -129,6 +130,14 @@ public async Task<IActionResult> ProcessConfirmEmployer(ConfirmEmployerViewModel
                     ModelState.AddModelError(member.Split('|')[0], member.Split('|')[1]);
                 }
 
+                foreach (var entry in ModelState)
```

Comment:
I can see why you've done this, but it seems a little flakey. I think I would look at creating your own custom exception that we throw when this sort of validation error occurs